project_electron/transfer_app/templates/transfer_app/transfer_detail.html

```diff
@@ -65,11 +67,39 @@ <h3 class="box-title">Error Messages</h3>
 							<th>Result</th>
 						</tr>
 					</thead>
+					<tbody>
+						{% for err in object.get_errors %}
+						<tr>
+							<td>{{ err.created_time }}</td>
+							<td>{{ err.code.code_desc }}</td>
+							<td></td>
+						</tr>
+						{% endfor %}
+					</tbody>
 				</table>
+				{% else %}
+				<p>No errors</p>
+				{% endif %}
+			</div>
+		</div>
+	</div>
+</div>
+
+
+{% if object.additional_error_info %}
```

Comment:
I have some ideas about how to improve this from a UI perspective, but I can do that when I deal with #53